pkgs/by-name/rq/rquickshare/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  appimageTools,
+  lib,
+  fetchurl,
+}:
+let
+  pname = "rquickshare";
+  version = "0.7.1";
+in
+appimageTools.wrapType2 {
+  inherit pname version;
+  src = fetchurl {
+    url = "https://github.com/Martichou/rquickshare/releases/download/v${version}/r-quick-share_${version}_amd64.AppImage";
+    hash = "sha256-716d7T4nbs/dDS4KVGTADCpLO31U8iq6hDVD+c7Ks1I=";
+  };
+
+  meta = {
+    description = "Rust implementation of NearbyShare/QuickShare from Android for Linux";
+    homepage = "https://github.com/Martichou/rquickshare";
+    changelog = "https://github.com/Martichou/rquickshare/blob/v${version}/CHANGELOG.md";
+    license = lib.licenses.agpl3Plus;
```

Comment:
https://github.com/Martichou/rquickshare/blob/master/LICENSE  

It's `gpl3Plus`

Reply:
Applied suggestion/fixes!

Reply:
It needs to replace `Exec=r-quick-share` with `Exec=rquickshare` in the desktop entry

Reply:
Apologies for keeping you waiting. Patched the desktop entry.

Should I also rename the desktop entry file from `r-quick-share.desktop` to `rquickshare.desktop`?

Reply:
> `r-quick-share.desktop` to `rquickshare.desktop`

It's okay to leave it untouched

And it's now `Icon=rquickshare %u`, so the icon is broken. You might want to replace that line `Exec=r-...`, not `r-quick-share`

Reply:
Should be fixed now